index.html

```diff
@@ -43,16 +46,13 @@ <h1 class="head">TED'S PORTFOLIO</h1>
           <nav class="siteLinks">
             <div class="icon-menu"><span>NAV</span></div>
             <ul>
-              <li><a class="siteLink tab" href="#">Home</a></li>
-              <li><a class="siteLink tab" href="#">About</a></li>
-              <li><a class="siteLink" href="#">Link 3</a></li>
-              <li><a class="siteLink" href="#">Link 4</a></li>
-            </ul>
-          </nav>
-          <nav class="socialLinks">
-            <ul>
-              <li><a class="socialLink" href="#"><img class="socialImgLink" src="vendor/images/FB-f-Logo__blue_29.png"/></a></li>
-              <li><a class="socialLink" href="#"><img class="socialImgLink" id="gitIcon" src="vendor/images/GitHub-Mark-Light-32px.png"/></a></li>
+              <li><a class="siteLink tab" href="#">Home</a></li><!--DeleteSpace
+              --><li><a class="siteLink tab" href="#">About</a></li><!--DeleteSpace
+              --><li><a class="siteLink" href="#">Link 3</a></li><!--DeleteSpace
+              --><li><a class="siteLink" href="#">Link 4</a></li><!--DeleteSpace
+              --><li><nav class="socialLinks">
```

Comment:
Wrap the social link icons in a nested ul, consider switching them to font icons for greater formatting control.